src/cmd/vendor.rs

```diff
@@ -433,8 +433,8 @@ pub fn diff(
                 Error::chain(
                     format!(
                         "Failed to copy {} to {}.",
-                        link_to.to_str().unwrap(),
                         link_from.to_str().unwrap(),
+                        link_to.to_str().unwrap(),
```

Comment:
I don't think this change here is necessary, as for the `diff` function, the changes are being copied from the `link_to` to the `link_from` (as seen on line 432).

Reply:
Force pushed a new commit without this change.